content/en/metrics/overview.md

```diff
@@ -0,0 +1,150 @@
+
+### Configurable Processing
+
+The **Configurable Processing** column lists the various advanced configuration options that you can use to increase the value of your metrics. Click on any of the options for more information and a link to the relevant configuration screen.
+
+{{% collapse-content title="Optimize your custom metrics costs with Metrics without Limits™" level="h4" %}}
+[Metrics without Limits™][19] helps you manage custom metric costs by indexing only the metric tags most valuable to your organization. Your use of Metrics without Limits™ is reflected in the top section of the Overview page as **Selected Metrics**. For more information about managing your custom metrics costs, see [Best Practices for Custom Metrics Governance][22].
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Enable percentiles on distribution metrics" level="h4" %}}
+[Distribution metrics][20] with percentiles enabled provide you with globally accurate percentiles calculated server-side from all hosts, so you can measure statistical distributions across your entire distributed infrastructure.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Metrics from other Datadog products" level="h4" %}}
+Some products incorporate standard metrics to surface insights out of the box (for example, APM). 
+
+##### Logs
+
+[Generate custom metrics from ingested logs][14] to summarize data from all the logs ingested by Datadog. This allows you to visualize and alert on log data that's important to your environment, even if the logs are not indexed for long-term search.
+
+##### APM
+
+[Generate custom metrics from your ingested spans][15] to visualize anomalies and trends across any parameters that are important to your business context.
+
+##### Real User Monitoring (RUM)
+
+[Generate custom metrics from RUM events][16] to summarize the data from your RUM events, so you can visualize and alert on the user behaviors that are most impactful to your organization.
+
+##### Processes
+
+[Generate custom process-based metrics][17] to monitor the resource consumption of your processes, as well as any other process-related behavior that might be important to your business needs.
+
+##### Events
+
+[Generate custom event-based metrics][18] for visibility into monitor alerts or any other event-based data ingested by Datadog.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Ingest historical metrics" level="h4" %}}
+[Historical Metrics Ingestion][21] enables you to ingest metric values with timestamps older than one hour from the time of submission.
+{{% /collapse-content %}} 
+
+### Available Metrics
+
+The **Available Metrics** column breaks down your total metric volume by standard and custom metrics over the past month. If you're interested in managing your custom metrics volume, see the [Best Practices for Custom Metrics Governance page][25] and the [metrics volume management page][26] for more detailed insights.
```

Comment:
```suggestion
The **Available Metrics** column breaks down your total metric volume by standard and custom metrics over the past month. If you're interested in managing your custom metrics volume, see the [Best Practices for Custom Metrics Governance page][25] and the [Metrics Volume Management page][26] for more detailed insights.
```

Reply:
All Metrics pages - Metrics Explorer, Metrics Overview, Metrics Summary have been capitalized in most existing metrics doc that I've written previously... (Metrics Volume is shorthand for Metrics Volume Management and is called out in our volume docs 
<img width="917" alt="Screenshot 2024-11-13 at 12 35 16 PM" src="https://github.com/user-attachments/assets/7f109357-0539-4fa7-af88-d1c6711d24b5">